website/components/Tile/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { RepeatIcon } from '@chakra-ui/icons'
+import {
+  Box,
+  SimpleGrid,
+  HStack,
+  Text,
+  useBreakpointValue,
+  useColorMode,
+} from '@chakra-ui/react'
+import { Link } from '@opengovsg/design-system-react'
+import { ThumbDownIcon } from '../ThumbDownIcon'
+import { Rec } from './Rec'
+import { useRecs } from './useRecs'
+
+const sentenceCase = (s: string) => s[0] + s.slice(1).toLowerCase()
+
+export const Tile = ({ title }: { title: string }) => {
+  const { currentRec, handleClick, isLoading } = useRecs(title)
+  const { colorMode } = useColorMode()
+  const isLight = colorMode === 'light'
+  const isMobile = useBreakpointValue({ base: true, xs: true, md: false })
+
+  return (
+    <SimpleGrid columns={1} templateRows='1fr 5fr' h='60vh'>
+      <HStack w='100%' maxW='30rem' justifyContent={'space-between'}>
+        <Text textStyle={isMobile ? 'display-2-mobile' : 'display-2'}>
+          {sentenceCase(title)}
+        </Text>
+        <HStack spacing='1rem'>
+          <Link onClick={() => handleClick('increment_downvotes')}>
+            <ThumbDownIcon
+              color={isLight ? 'black' : 'white'}
+              fontSize={isMobile ? '1.5rem' : '2rem'}
+            />
+          </Link>
+          <Link onClick={() => handleClick('increment_clicks')}>
```

Comment:
`increment_clicks` is meant for when the user clicks on the link to check out the recommendation. I think you did accidentally remove the link that was wrapping the tile? Could you add it back in and trigger `increment_click` for when they click the tile to get off to the external url. 

For this one we just need to call `onClick={() => handleClick()}`

Reply:
thanks for the catch! addressed in d1c54b6. unfortunately I didn't find an easy way to wrap the whole Tile in a `Link` without messing up the styling, so I've wrapped the image and the text each in their own `Link`. I've also edited `handleClick` to prevent moving on to the next rec when clicking on the link.